llvm-runtimes/openmp/openmp-15.0.7-r6.ebuild

```diff
@@ -76,6 +76,14 @@ pkg_setup() {
 	use test && python-any-r1_pkg_setup
 }
 
+src_prepare() {
+	cmake_src_prepare
+
+	# upstream forgot to respect LLVM_CMAKE_DIR when finding LLVM
+	sed -e 's/^\(\s*find_package(LLVM\b.*\))$/\1 HINTS "${LLVM_CMAKE_DIR}")/' \
```

Comment:
This should also become unnecessary if you use `-DCMAKE_PREFIX_PATH`.

Reply:
I'll give it a try.